lib/csv_to_popolo.rb

```diff
@@ -406,6 +406,8 @@ def cell_values(key, separator=nil)
           TwitterUsernameExtractor.extract(v) rescue nil
         elsif key == :facebook
           "https://facebook.com/#{FacebookUsernameExtractor.extract(v)}" rescue nil
+        elsif key == :email
+          v = v.gsub('mailto:','')
```

Comment:
Assigning back to `v` here is a little odd, and makes this harder to read as it's different behaviour from the other parts of the same `if` (leading an average reader to wonder _why_, and what's different about this one, when the answer is really 'nothing')